controlpanel/frontend/views/datasource.py

```diff
@@ -213,13 +213,28 @@ def get_success_url(self):
 
 
 class GrantAccess(LoginRequiredMixin, PermissionRequiredMixin, CreateView):
+    context_object_name = 'users3bucket'
     form_class = GrantAccessForm
     model = UserS3Bucket
     permission_required = 'api.create_users3bucket'
+    template_name = 'datasource-access-grant.html'
 
     def get_form_kwargs(self):
         return FormMixin.get_form_kwargs(self)
 
+    def get_context_data(self):
+        context = super().get_context_data()
+        bucket = get_object_or_404(S3Bucket, pk=self.kwargs['pk'])
+        context['bucket'] = bucket
+        access_group = bucket.users3buckets.all().select_related('user')
+        member_ids = [member.user.auth0_id for member in access_group]
+        context['access_group'] = access_group
```

Comment:
Does `access_group` need to go in the `context`? Is it used by the template, etc...?

Reply:
Good catch 👍